front/lib/api/assistant/conversation.ts

```diff
@@ -767,6 +768,7 @@ export async function* retryAgentMessage(
       {
         status: "created",
         agentConfigurationId: messageRow.agentMessage.agentConfigurationId,
+        agentConfigurationVersion: 0,
```

Comment:
?

Reply:
my bad this was for testing forgot to remove